migrations/V122__Update_PlanBasics_ModelType_MultiSelect_And_Other_Text.sql

```diff
@@ -0,0 +1,40 @@
+ALTER TYPE MODEL_TYPE RENAME TO MODEL_TYPE_OLD;
+
+-- Add the 'OTHER' and 'MANDATORY_REGIONAL_OR_STATE' values to the old type so we can migrate 'TBD' and 'MANDATORY'
+-- selections later in this migration
+ALTER TYPE MODEL_TYPE_OLD ADD VALUE 'OTHER';
+ALTER TYPE MODEL_TYPE_OLD ADD VALUE 'MANDATORY_REGIONAL_OR_STATE';
+
+
+CREATE TYPE MODEL_TYPE AS ENUM (
+  'VOLUNTARY',
+  'MANDATORY_REGIONAL_OR_STATE',
+  'MANDATORY_NATIONAL',
+  'OTHER'
+);
+
+COMMIT;
+
+ALTER TABLE plan_basics
+  ADD COLUMN model_type_other zero_string;
+
+-- If 'TBD' is selected we want to migrate it into the 'OTHER' note field and set our new
+-- model type selection to 'OTHER'
+UPDATE plan_basics
+  SET model_type_other = 'TBD',
+      model_type = 'OTHER'
+  WHERE 'TBD' = model_type;
+
+-- If 'MANDATORY' is selected we want to migrate it to either 'MANDATORY_REGIONAL_OR_STATE' or 'MANDATORY_NATIONAL'
+-- As prod data is evenly split, we simply choose one
+UPDATE plan_basics

```

Comment:
@OddTomBrooks, are these options going to be manually updated in PROD? This is what I see in JIRA. 

I don't see instructions for how to handle `TBD`, but your approach seems logical. Did you confirm this with product?

```
Historical data migration for those that had "Mandatory" selected:
CJR 3-Year Extension << update to Mandatory regional or state
Expanded Home Health Value-Based Purchasing Model << update to Mandatory national
Increasing Organ Transplant Access (IOTA) Model << update to Mandatory regional or state
Most Favored Nation Model (MFN) << update to Mandatory national

```

Reply:
Yes exactly. Gary and I discussed this and I'll reach out to Jeff for confirmation.

Reply:
I reached out to Jeff regarding this. He instructed that from a product perspective it's best to simply drop the TBD option, so we are nulling it out.

Reply:
Thanks Tom, also to confirm, are the individual models that need to  be `MANDATORY_NATIONAL` being manually updated? (And who is doing that if so?)

Reply:
I am doing that manually via database edits once the merge goes through. I didn’t feel the specificity of model names as a migration policy or sharing model data in GitHub through a migration was appropriate. It’s only a few rows, and I minimized manual audits by having a default case migration.

Reply:
By merge, I mean once the feature is deployed.

Reply:
Perfect, yeah that works for me. I just wanted to make sure I understood what the approach is. But I agree, these can be handled in a one off manor. It might be less complex just to resolve those few in the UI